main/parse.c

```diff
@@ -126,12 +126,12 @@ static parserDefinition *FallbackParser (void);
 static parserDefinition *CTagsParser (void);
 static parserDefinition *CTagsSelfTestParser (void);
 static parserDefinitionFunc* BuiltInParsers[] = {
-	CTagsParser,				/* This must be first entry. */
-	FallbackParser,				/* LANG_FALLBACK */
-	CTagsSelfTestParser,
 #ifdef EXTERNAL_PARSER_LIST
 	EXTERNAL_PARSER_LIST
 #else  /* ! EXTERNAL_PARSER_LIST */
+	CTagsParser,				/* This must be first entry. */
```

Comment:
I am wondering the meaning of the comment. I cannot remember why it must be first.
I will try to remember.

Reply:
As far as reading b9636b4db0e6c3e3f42ed1bd4be16b5c270315eb, there is no reason being CTagsParser at the fist of the list. Actually, it was not the first on the list. However, I would like to test before merging this pull request.

Reply:
At least for Geany it works without any problem when all these parsers are skipped.